### ISSUE TITLE
CFB-1: ACVP server cannot process response

### ISSUE THREAD

**Issue by smuellerDD** [usnistgov/ACVP#403]
The ACVP server verdict for an AES-CFB-1 contains:

```
[ {
  "acvVersion" : "0.4"
}, {
  "results" : {
    "vsId" : 8173,
    "disposition" : "error",
    "message" : "validation executable failed for: 8173"
  }
} ]
```

**Comment by celic** [comment]
AES-CFB-1 is a bit wonky. It expects the plaintext and ciphertext to be single bits, either 1 or 0 (as strings). I believe this has been fixed in v0.5, but for v0.4, please use this format.

**Comment by smuellerDD** [comment]
Am Mittwoch, 20. Juni 2018, 18:04:15 CEST schrieb Chris Celi:

Hi Chris,

> AES-CFB-1 is a bit wonky. It expects the plaintext and ciphertext to be
> single bits, either 1 or 0 (as strings). I believe this has been fixed in
> v0.5, but for v0.4, please use this format.

Oh, I thought that was fixed -- or can I point to version 0.5 in my reply?

Ciao
Stephan

**Comment by celic** [comment]
We haven't released v0.5 yet, so you cannot point to it. (Soon!)

**Comment by celic** [comment]
This problem is fixed in v0.5 and you will use normal hexadecimal instead of the individual bits then. For now, use "1" and "0".
